RELEASING.md

```diff
@@ -0,0 +1,39 @@
+# Scala.js Release Process
+
+1. Clean up bugs:
+    - Make sure all bugs assigned to the milestone are closed.
+    - Make sure all fixed bugs are assigned to the milestone ([this query][1]
+      should not return anything).
+1. Create a "Version x.y.z." commit ([example][2]) and push it to a branch on
+   your fork.
+1. Ping people on the commit for review.
+1. Once you have LGTM, push to master (do *not* create a merge commit).
+1. Testing (post results as comments to commit):
+    - Nightly
+    - Weekly
+    - [Manual testing][3]
+1. If all tests pass, tag the commit with the release version.
+1. Perform [manual testing][3] that needs the tagging (source maps).
+1. Publish:
+    - Sonatype, bintray (`./script/publish.sh`)
+    - CLI to website (simple `scp`)
+    - Docs to website: Use
+      `~/fetchapis.sh <full sjs version> <binary sjs version>` on the webserver
+      once artifacts are on maven central.
+1. Once artifacts are on maven central, create a "Towards x.y.z." commit
+   ([example][4]). Same procedure for push to master than the "Version x.y.z."
+   commit.
+1. Prepare release announcement, taking the last one as model ([example][5]).
+1. When merging the release announcement PR (after proper review):
+    - Update the latest/ URLs (use `~/setlatestapi.sh <full sjs version>` on
+      webserver)
+    - Announce on Twitter using the @scala_js account
+    - Announce on [Gitter](https://gitter.im/scala-js/scala-js)
+    - Announce on the mailing list (scala-js@googlegroups.com) 
+    - Cross-post to scala-announce (scala-announce@googlegroups.com)
+
+[1]: https://github.com/scala-js/scala-js/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20no%3Amilestone%20-label%3Ainvalid%20-label%3Aduplicate%20-label%3Aas-designed%20-label%3Aquestion%20-label%3Awontfix
```

Comment:
https://github.com/scala-js/scala-js/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20no%3Amilestone%20-label%3Ainvalid%20-label%3Aduplicate%20-label%3Aas-designed%20-label%3Aquestion%20-label%3Awontfix%20-label%3A%22can%27t%20reproduce%22%20-label%3A%22separate%20repo%22 to ignore "separate repo" and "can't reproduce" as well.